.github/actions/integration-tests/run-ci-stage1

```diff
@@ -396,6 +396,19 @@ for userenv in ${CI_ACTIVE_USERENVS}; do
                     run_cmd "crucible run iperf,uperf --tags run:2${TAGS} --num-samples ${CI_SAMPLES} --test-order r --mv-params ${SCRIPT_DIR}/mv-params/iperf-2.json,${SCRIPT_DIR}/mv-params/uperf-2.json ${ENDPOINT_ARG}"
                     post_run_cmd
                 fi
+
+                # multibench all-in-one json test
+                RUN_FILE="${SCRIPT_DIR}/run-file/ci-${scenario}-run-file-${CI_ENDPOINT}-${userenv}.json"
+                if [ -f "${RUN_FILE}" ]; then
+                    source ${SCRIPT_DIR}/prepare-run-file 
+                    replace_value userenv ${userenv} '."endpoints"[0]."userenv" = $userenv'
```

Comment:
Based on the value of `RUN_FILE` set above this seems like it isn't needed...or am I missing something?

Reply:
-f checks if the file exists. In a previous action in crucible-ci, we copy all ci-* files from rickshaw/util/tests, so all files are copied there. If there is a file for a this scenario and userenv, then the ci job will run the runfile test.